src/phoebus_driver.cpp

```diff
@@ -148,6 +149,17 @@ TaskCollection PhoebusDriver::RungeKuttaStage(const int stage) {
     pmono_vol_red = monopole->MutableParam<MonoVolRed_t>("volumes_reducer");
   }
 
+  // Hack to update time for time-dependent geometries
+  auto geometry = pmesh->packages.Get("geometry");
+  auto &geom_params = geometry->AllParams();
+  // tm = SimTime field in EvolutionDriver. See parthenon/src/driver/driver.hpp
+  if (geom_params.hasKey("time")) {
+    const Real tstage = tm.time + robust::ratio((stage - 1) * tm.dt,
```

Comment:
is this right?  for our default form of RK2, the prime state is a t+dt not t+dt/2.

Reply:
Oh hmm we're using Heun's method. Maybe this is the right thing, under full generality...
```C++
const Real tstage = tm.time + (stage > 1 ? (integrator->beta[stage-2]) : 0)*tm.dt;
```
so for stage 1 it's just the current time. Then for subsequent stages, it's the time + `beta*dt` What do you think?

Reply:
yeah I think this is right